rust/src/settings.rs

```diff
@@ -436,6 +437,42 @@ pub struct ModelSettings {
     pub size: usize,
 }
 
+#[derive(Debug, Deserialize)]
+/// Metrics settings.
+pub struct MetricsSettings {
+    /// The URL address of the metrics store.
+    ///
+    /// # Examples
+    ///
+    /// **TOML**
+    /// ```text
+    /// [metrics]
+    /// store_url = "http://localhost:8086"
+    /// ```
+    ///
+    /// **Environment variable**
+    /// ```text
+    /// XAYNET_API__STORE_URL=http://localhost:8086
```

Comment:
Shouldn't these be `XAYNET__METRICS_FOO` instead?

Reply:
yes indeed, well caught!